src/main/java/org/opentripplanner/updater/vehicle_position/GtfsRealtimeHttpVehiclePositionSource.java

```diff
@@ -26,23 +27,26 @@ public class GtfsRealtimeHttpVehiclePositionSource implements VehiclePositionSou
    */
   private final URI url;
 
-  private final Map<String, String> DEFAULT_HEADERS = Map.of(
-    "Accept",
-    "application/x-google-protobuf, application/x-protobuf, application/protobuf, application/octet-stream, */*"
-  );
+  private final HttpHeaders DEFAULT_HEADERS = HttpHeaders
+    .of()
+    .add(
+      "Accept",
+      "application/x-google-protobuf, application/x-protobuf, application/protobuf, application/octet-stream, */*"
+    )
+    .build();
 
-  private final Map<String, String> headers;
+  private final HttpHeaders headers;
 
-  public GtfsRealtimeHttpVehiclePositionSource(URI url, Map<String, String> headers) {
+  public GtfsRealtimeHttpVehiclePositionSource(URI url, HttpHeaders headers) {
     this.url = url;
-    this.headers = MapUtils.combine(headers, DEFAULT_HEADERS);
```

Comment:
Imposible to override "Accept" header.

Reply:
With the solution I've implemented it is now possible: https://github.com/opentripplanner/OpenTripPlanner/pull/5035/commits/1a523b7ea13230d23345824aee9c32f720cf963f